runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsOverlay.java

```diff
@@ -70,6 +71,10 @@ public Dimension render(Graphics2D graphics)
 			{
 				polygon = ((GameObject) object).getConvexHull();
 			}
+			else if (object instanceof DecorativeObject)
+			{
+				polygon = ((DecorativeObject) object).getConvexHull();
```

Comment:
This will fail on diagonal walls with back to back decorative objects.
If you want to see this behaviour yourself check Juliets house, there is a painting on a diagonal wall on the ground floor.

Abex was looking into this.

Reply:
Is this fixed now from #7824 ?

Reply:
with #7824 this just needs to render both convex hulls